### PR TITLE
feat: implement p2p querier

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/90

Contributes to: https://github.com/celestiaorg/orchestrator-relayer/issues/108
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by SweeXordious** [comment]
Depends on: https://github.com/celestiaorg/orchestrator-relayer/pull/110

File: p2p/querier.go [hunk 1]
```diff
@@ -2,61 +2,258 @@ package p2p
...
+				"total_power",
+				currThreshold,
+				"number_of_confirms",
+				len(confirms),
+				"missing_confirms",
+				len(valset.Members)-len(confirms),
+			)
+		}
+		// TODO: make the timeout configurable
+		time.Sleep(10 * time.Second)
```
Comment (evan-forbes) [inline p2p/querier.go:177]: what is this for?

**Review by evan-forbes — COMMENTED** [review]
mostly LGTM, had one blocking question though

File: p2p/querier.go (hunk 1 above)
Comment (SweeXordious) [reply p2p/querier.go:177]: Just to keep a delay between the different calls to querying 2/3. I wanna change it with a ticker, what do you think? every 10sec

Comment (evan-forbes) [reply p2p/querier.go:177]: I see, a ticker could be more explicit yeah

**Review by evan-forbes — DISMISSED** [review]

File: p2p/querier.go (hunk 1 above)
Comment (SweeXordious) [reply p2p/querier.go:177]: https://github.com/celestiaorg/orchestrator-relayer/issues/119

**Comment by SweeXordious** [comment]
@evan-forbes Can you re-approve please? just resolved the conflicts

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**